game-core/src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -190,10 +190,11 @@ private void testEmail() {
   public boolean isBeanValid() {
     final boolean hostValid = validateTextFieldNotEmpty(host, hostLabel);
     final boolean portValid = validateTextField(port, portLabel, new IntegerRangeValidator(0, 65635));
-    // boolean loginValid = validateTextFieldNotEmpty(login, loginLabel);
-    // boolean passwordValid = validateTextFieldNotEmpty(password, passwordLabel);
```

Comment:
Wow, didn't expect validation code to exist already.
For some reason @veqryn commented out this code in 9a4ea25a7d9a46f929237a6af1b366e4cf1f6cbd

Reply:
I'm not sure @veqryn was the one to remove it.  From the commit message, it looks like the author was "qwertgold".  I'm assuming this was from a time when Chris was the only committer on SourceForge and other contributors would send him patches to integrate (ah, the good old days :smile:).

Reply:
Ah, right. 
I was just skimming through the diff to find the commit this was done in.
It sounds super annoying to have to apply patches manually 😅